decentraland/unity-explorer
# style: fix passport loading items mask

## What does this PR change?
This PR was created to fix the loading item mask compatibility. Instead of using soft mask now we are using the native one.
![image](https://github.com/user-attachments/assets/b21b5f36-4708-4d7f-980e-ad350a5aaa5d)

## How to test the changes?
1. Launch the explorer
2. open your profile or someone else's and check that while fetching the equipped items the loading animation is not visible outside the profile card container.
